### DISCUSSION TITLE
Support for nostr?

### DISCUSSION THREAD

**Discussion by ghost** [d99kris/nchat#79] [Q&A]
I use nchat as my Telegram client on my laptops and desktops all the time and I very much like it.  I believe that support for whatsapp is being added, so I guess nchat is flexible enough to support different messaging systems/protocols.

I also like nostr, but I can't find any client I like.  I'd love for nchat to support nostr as a client.  To me it'd be a great combination, the best client for the best protocol.

Any chances of that ever happening?  :-)

**Comment by d99kris** [comment]
Hi @alganza - glad you like `nchat`! 

> I believe that support for whatsapp is being added, so I guess nchat is flexible enough to support different messaging systems/protocols.

Yes, WhatsApp has already been added, and the design of nchat allows adding more protocols. 

> I also like nostr, but I can't find any client I like. I'd love for nchat to support nostr as a client. To me it'd be a great combination, the best client for the best protocol.
> Any chances of that ever happening? :-)

I'm afraid the likelihood of this is quite low. I myself is not using `nostr`, so I am not inclined to implement support and maintain it. But if someone would be willing to contribute implementation for it, and can help with related bugs, I would be open to consider including it in the `nchat` code base. However, the likelihood of someone contributing that is probably low.

**Reply by ghost** [reply]
Thank you for your answer!

Now that Jack Dorsey is supporting Nostr, both with his words and with his pocket (he's given fiatjaf 14 BTC, that is around $250K, to start funding development of relays and clients) and that Elon Musk has forbidden posting links to Nostr (the guy has no clue on how it works, obviously), everybody is hearing about nostr and hopefully somebody would be interested in adding support to it on nchat.  I wish I could do it, but even when apparently the protocol is very basic and simple to understand, my coding skills (almost as poor as Musk's)  aren't good enough to even think about it.
